src/internal/interfaces/Visualization.ts

```diff
@@ -209,6 +209,7 @@ export interface IBucketUiConfig {
     allowShowOnSecondaryAxis?: boolean;
     allowSelectChartType?: boolean;
     allowOptionalStacking?: boolean;
+    acceptLocationDisplayForm?: boolean;
```

Comment:
We have some conflicting name here `allow...` vs `allows...`
https://github.com/gooddata/gooddata-react-components/blob/f0e7446990718924a2e4ce0b692c20ac33292a31/src/internal/interfaces/Visualization.ts#L201

In this case, I prefer `accepts...`.
What do you think?

Reply:
I would go with `accepts` as well. Thanks